lib/temporal.rb

```diff
@@ -3,6 +3,25 @@
 
 require 'temporal/connection'
 require 'temporal/version'
+require 'socket'
 
 module Temporal
+  CLIENT_NAME = 'temporal-ruby'.freeze
+
+  # TODO: Implement me
+  def self.build_id
```

Comment:
This probably doesn't need to be exposed via `Temporal.build_id` since it can be overwritten per Worker invocation. We might wanna follow Python's lead here and do something similar to https://github.com/temporalio/sdk-python/blob/main/temporalio/worker/_worker.py#L547